### DISCUSSION TITLE
Christopher "vjeux" Chedeau

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#257] [Announcements]
- https://blog.vjeux.com
- https://github.com/vjeux
- https://twitter.com/Vjeux

## Topics

- React Native
- Prettier
- Excalidraw
- "CSS-in-JS", Yoga
- Pokémon Side Project
- Video Editor Side Project

## Questions

- [x] You've worked at Meta on a crazy amount of influential projects. What projects were those? How do you like the OSS culture of Facebook and how do you think the future of that looks like?
- [ ] ~You recently went on a journey through the blue/red Pokemon map? Can you tell out listeners about the project, why you did it, and any fun things you discovered while writing the code?~
- [x] You created prettier, arguably saving millions of years of developers arguing over formatting. Thanks for that! It changed the way write code (and how fast I can write it). Prettier recently has a pretty cool competition with a big prize, could you fill us in about that?
- [x] There is a trend of rewriting tools for JS in faster languages. What do you think about that?
- [x] You created create-react-app and probably ushered in an age of SPA. What do you think of the current tools to create react apps?
- [x] You tout yourself as the creator of "css-in-js". Now that the dust has settled how do you think that affected the industry? The idea sparked countless libraries trying to solve it in the best way, but it took a long time.
- [x] Of what we have what would you use? Is it just style-x?
- [ ] ~Speaking of react you helped to create react native. Being able to code for different platforms is arguably react's killer feature. Do you think there is room for other "react-x" platforms? What platform do you think might catch on? (I saw a react-native-vision 🤯 )~
- [x] You seems to choose particularly high impact and well received projects to work on. Prettier ended debates on code formatting, excalidraw brought diagraming to all, react-native and yoga made JS devs have super powers. What do you plan to conquer next? I think I saw you building a video editor.
- [x] Excalidraw started as a pure OSS tool, but has evolved into something more. How has that gone and is the project sustainable?
- [x] You seem to have the ability to pick high impact OSS projects. What do you look for when choosing  a "good" project to work on?
- [x] What are you working on next?
- [x] With the advent of AI tools our jobs might go the way of the dinosaur. What do you think the future looks like for our industry? For our jobs?

**Comment by hipstersmoothie** [comment]
My Tooltips:

https://www.npmjs.com/package/workerpool
https://github.com/sindresorhus/promise-fun
https://react-postgres-components.vercel.app/

**Comment by just-be-dev** [comment]
Tooltips:
- https://hyperscript.org/
- https://github.com/launchbadge/sqlx
